go.mod

```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/cyberark/conjur-api-go v0.6.0
-	github.com/cyberark/conjur-authn-k8s-client v0.19.0
+	github.com/cyberark/conjur-authn-k8s-client v0.19.1
```

Comment:
i think that we can add a CHANGELOG entry for this change and link to the released version page of the authn-client so the new version's CHANGELOG is available and it's clear what user-facing changes happened in this version.

Reply:
Will add an entry.